src/style.css

```diff
@@ -0,0 +1,93 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  color: rgb(255 255 255);
+  font-size: 100px;
```

Comment:
specify the `font-family` too 😉
it has some unusual font on my machine

<img width="581" alt="image" src="https://user-images.githubusercontent.com/55802630/173016907-c61263af-751e-40c7-8105-c4cc73bc66d8.png">